lib/phlex/svg/standard_elements.rb

```diff
@@ -74,175 +74,175 @@ module Phlex::SVG::StandardElements
 	register_element :ellipse, tag: "ellipse"
 
 	# @!method feBlend(**attributes, &content)
-	# 	Outputs a `<feBlend>` tag.
+	# 	Outputs an `<feBlend>` tag.
```

Comment:
I guess this is pronounced `eff-ee-blend` rather than `fee-blend`. Good catch.

Reply:
I've never used it, so I could be wrong 🤷‍♂️ but I would have naively read it "eff-ee"